.circleci/config.yml

```diff
@@ -0,0 +1,44 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
```

Comment:
I wonder why all these comments that do apparent things were left here, any reason?

Reply:
Will remove them